images_config/ks-robot-master.cfg

```diff
@@ -106,71 +106,72 @@ update-grub
 fi
 %end
 
-##### UPGRADE KERNEL #################################################
+################ UPGRADE KERNEL ################ 
 %post --interpreter=/bin/bash
 apt-get install --install-recommends linux-generic-lts-xenial -y --force-yes
 %end
 
-################## INSTALL GRAPHICAL INTERFACE ############
+################ INSTALL GRAPHICAL INTERFACE ################ 
 %post --interpreter=/bin/bash
 apt-get install lightdm -y -f
 dpkg-reconfigure lightdm
 %end
 
-####################INSTALLING ROS#######################################     
-%post --interpreter=/bin/bash
-!/bin/bash
-sh -c 'echo "deb http://packages.ros.org/ros/ubuntu trusty main" >> /etc/apt/sources.list.d/ros-latest.list'
-wget http://packages.ros.org/ros.key -O - | apt-key add - 
-apt-get update
-apt-get install ros-indigo-roscpp -y
-apt-get install ros-indigo-rosbash -y
-apt-get install ros-indigo-ros -y
-apt-get install ros-indigo-care-o-bot-robot -y
+################ ADD USERS ################ 
+%post --interpreter /bin/bash
+echo "::Adding users and groups ::"
+echo "session	required	pam_mkhomedir.so " >> /etc/pam.d/common-session
```

Comment:
By using above solution its not solved, without this line  **echo "session	required	pam_mkhomedir.so " >> /etc/pam.d/common-session** we are unable to create robot user, and when we use the line it will create after first session of login. I did few more changes in nfs module will update you when I'm done with booting.